dockers/docker-sonic-telemetry/docker-telemetry-entry.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+while true; do
```

Comment:
This loop seems a hack to keep continer running but acting as a disabled. Could you improve the design in future iteration?